### ISSUE TITLE
Genome assembly of aneuploid plant species 

### ISSUE THREAD

**Issue by sandipmkale** [tangerzhang/ALLHiC#125]
Hello,  i am working on a plant species which is highly heterozygous and contains 3 trisomic chromosomes. We have both HiFi and HiC data. Is it possible to generate phased assembly using all hic pipeline. 

Thanks alot 

Sandip

**Comment by tangerzhang** [comment]
Yes, It should be OK to generate the phased assembly using ALLHiC pipeline.
